src/containers/App.js

```diff
@@ -76,7 +76,7 @@ export default class App extends React.Component {
   render () {
     const {
       children, community, dispatch, leftNavIsOpen, notifierMessages, isMobile,
-      showModal
+      showModal, route: { socket }
```

Comment:
`socket` seems like a good candidate to include in `childContextTypes`, since it's not specific to `ModalWrapper`.